### DISCUSSION TITLE
Esc can't dismiss the 'ctrl-k' search.

### DISCUSSION THREAD

**Discussion by ranbims** [community/community#6990] [Other Feature Feedback, Questions, & Ideas] [answered]
I like the new feature. 'ctrl-k' to search on input commands. But I didn't know how to dismiss the search bar by keyboard. I have to 'click outside' by my mouse. It is wired to launch this by keyboard shortcuts but dismiss it with a mouse click.

**Comment by gaknoll** [comment]
@ranbims we have a fix for this coming soon!

**Comment by gaknoll** [comment, marked as answer]
@ranbims `Esc` works for a majority of users, but unfortunately certain browser extensions override the keyboard shortcut and prevent you from closing the command palette using `Esc`. We did however just ship an update that allows for you to both open and close the command palette with `Ctrl k` or `Command k` so that you don't have to take your hands off the keyboard.

**Reply by ranbims** [reply]
@gaknoll Thanks! That works for me!

**Comment by dromer** [comment]
I do not like this feature at all. `ctrl+k` in most web-browsers defaults to the search bar. The only way to get to the search bar now is to first `alt+d` (address bar) and _then_ `ctrl+k` to search.

Please allow for a way to disable this key-binding as it prohibits any normal browsing for me.

**Reply by ranbims** [reply]
@dromer If you are using Chrome or Edge, you can use `ctrl+e` instead.

**Reply by dromer** [reply]
I do not. Have instead completely disabled this feature.
